stable/cluster-autoscaler/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 0.14.2
+version: 0.15.0
```

```suggestion
version: 2.0.0
```

Since this radically changes the PSP, and might not match people's expectations, I think a major version bump is in order, and _master_ is at `1.1.0` as I type this.